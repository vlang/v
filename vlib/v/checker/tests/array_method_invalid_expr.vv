runes := [`a`, `b`, `c`, `a`]
dump(runes.all(`a`))
dump(runes.any(`a`))
dump(runes.count(`a`))
dump(runes.filter(`a`))
