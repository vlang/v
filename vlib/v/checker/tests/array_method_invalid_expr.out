vlib/v/checker/tests/array_method_invalid_expr.vv:2:16: error: invalid expression, expected infix expr, lambda or function
    1 | runes := [`a`, `b`, `c`, `a`]
    2 | dump(runes.all(`a`))
      |                ~~~
    3 | dump(runes.any(`a`))
    4 | dump(runes.count(`a`))
vlib/v/checker/tests/array_method_invalid_expr.vv:3:16: error: invalid expression, expected infix expr, lambda or function
    1 | runes := [`a`, `b`, `c`, `a`]
    2 | dump(runes.all(`a`))
    3 | dump(runes.any(`a`))
      |                ~~~
    4 | dump(runes.count(`a`))
    5 | dump(runes.filter(`a`))
vlib/v/checker/tests/array_method_invalid_expr.vv:4:18: error: invalid expression, expected infix expr, lambda or function
    2 | dump(runes.all(`a`))
    3 | dump(runes.any(`a`))
    4 | dump(runes.count(`a`))
      |                  ~~~
    5 | dump(runes.filter(`a`))
vlib/v/checker/tests/array_method_invalid_expr.vv:5:19: error: invalid expression, expected infix expr, lambda or function
    3 | dump(runes.any(`a`))
    4 | dump(runes.count(`a`))
    5 | dump(runes.filter(`a`))
      |                   ~~~
